controllers/storagecluster/storageclasses.go

```diff
@@ -270,9 +271,10 @@ func newCephOBCStorageClassConfiguration(initData *ocsv1.StorageCluster) Storage
 // newStorageClassConfigurations returns the StorageClassConfiguration instances that should be created
 // on first run.
 func (r *StorageClusterReconciler) newStorageClassConfigurations(initData *ocsv1.StorageCluster) ([]StorageClassConfiguration, error) {
+	allowVolumeExpansionForBlockPoolStorageClass := true
```

Comment:
```suggestion
	allowVolumeExpansion := true
```

Reply:
There are two storage classes created below - cephFS and blockPool.
Hence added a suitable meaningful name to identify for which we are setting it here. We don need volume expansion variable for cephfs